docs/docs/identity-providers/readme.md

```diff
@@ -8,23 +8,14 @@ description: >-
 
 # Identity Provider Configuration
 
-This article describes how to configure Pomerium to use a third-party identity service for single-sign-on.
+Pomerium helps provide a zero-trust security model by verifying that every request to your upstream applications are from an authenticated and authorized user. User and group information is sourced from your Identity Provider (**IdP**). Pomerium communicates with IdPs following the [OpenID Connect][openid connect] (**OIDC**) standard for user identity, and a service account and/or API token for group/directory information. The steps for integrating Pomerium with an IdP are specific to each provider, but they generally share the same base requirements:
 
-There are a few configuration steps required for identity provider integration. Most providers support [OpenID Connect] which provides a standardized identity and authentication interface.
+- A **[Redirect URL](https://www.oauth.com/oauth2-servers/redirect-uris/)** pointing back to Pomerium. For example, `https://${authenticate_service_url}/oauth2/callback`.
+- A **[Client ID]** and **[Client Secret]**.
+- A **[Service Account]** for additional IdP Data. This is enables Pomerium administrators to write policies around groups, or any other data that doesn't uniquely identify an end-user, as defined in the IdP.
```

Comment:
```suggestion
- A **[Service Account]** for additional IdP Data. This enables Pomerium administrators to write policies around groups, or any other data that doesn't uniquely identify an end-user, as defined in the IdP.
```

Reply:
I'm not sure what
> or any other data that doesn't uniquely identify an end-user, as defined in the IdP
means

Could you elaborate?

Reply:
I was just covering my bases for what I didn't yet know Pomerium could do. Sounds like I should remove it.